setup.py

```diff
@@ -31,7 +31,7 @@
     # Use with pip install . to install from source
     'install': [
         'cartopy',
-        'cf-units>=3.0.0',
+        'cf-units>=3.0.0,!=3.0.1.post0',
```

Comment:
can you add a short comment here why the blocked post0 pls - so we know when/why to remove it in the future

Reply:
done